### DISCUSSION TITLE
Create Flatpak based on binary .tar.gz and submit to Flathub for easier installation.

### DISCUSSION THREAD

**Discussion by kevinbburns** [microsoft/azuredatastudio#22266] [General]
I would like to know if I can package the binaries into a flatpack to submit to FlatHub to be deployed via their Software hub for easy installation. They need confirmation via a URL before I can apply.

**Comment by erinstellato-ms** [comment]
Hi @kevinbburns - I'm not familiar with FlatHub and would like to understand exactly what would be needed on the ADS side. Do you have documentation or a link you could share?  Also, is this something that you would then expect to be updated for every release, and would that be something you would plan to handle?  Thanks.

**Reply by kevinbburns** [reply]
Hey,

I put a few links below; Flatpak's are similar to Snap images, just a neat way to distribute Linux applications via a "store," similar to Microsoft's Windows store. Nothing would be required from the ADS side, and I would take on the responsibility of creating the packages, getting them approved, and updating them since this is an application I use regularly. It just caught my eye in the "requests" section that this was a highly requested application, and I wanted to chip in where I could. 

Creating a FlatPak:
https://docs.flatpak.org/en/latest/first-build.html

Flathub:
https://flathub.org/home

**Comment by erinstellato-ms** [comment]
@kevinbburns Is there a way to note, on FlatHub, that the packaging is managed by you and not by Microsoft?  I'm still investigating if this is ok to do from a legal perspective (beyond my ability to answer!), but wanted to know that as well.  Thanks!
